README.md

```diff
@@ -24,9 +24,22 @@ Default Docker image is [yegor256/rultor](https://registry.hub.docker.com/u/yego
 
 TBD... _product statement_
 
-## What Problem Does Rultor Solve?
+## What Problems Does Rultor Solve?
 
-TBD... _stakeholders and needs_
+Automated build scripts have been around for some time. Rultor attempts to
+tackle the problems those scripts do not.
```

Comment:
@mafagafogigante I don't think you described here these problems that are not tackled by build scripts. Or you meant Docker containers?